### DISCUSSION TITLE
Top mounted cam looking down

### DISCUSSION THREAD

**Discussion by kuumaur** [blakeblackshear/frigate#4662] [Ask A Question]
Hi, I have an issue with my new Reolink Duo 1 poe. I have mounted that camera and it "looks" down to the driveway and the path to main door. The footage is good and the motion boxes register motion, but no event is triggered.
My 2 other cameras do not show this issue, they work as expected. Could it be the case, that frigate cant recognize objects from a top-to-down view?
Thanks in advance
-kuumaur

**Comment by NickM-27** [comment]
Would recommend watching the debug view with bounding boxes, do they get detected at all?

Top down usually benefits from high resolutions than 640x480 for example. Sharing your current config might help.

**Reply by kuumaur** [reply]
Yes, detection is working, at least for persons. Smaller objects like dogs or cats are not detected. When I say detection is working, then it means, that detection is working but recognizing is not working. SO see the bounding boxes around the object, but it did not say "person" or something else.

**Comment by kuumaur** [comment]
Here is a part of my config for one camera:

cameras:
  carport:
    mqtt:
      required_zones:
        - zone_0
    rtmp:
      enabled: false
    ffmpeg:
      hwaccel_args:
        - -hwaccel
        - vaapi
        - -hwaccel_device
        - /dev/dri/renderD128
        - -hwaccel_output_format
        - yuv420p
      inputs:
        - path: rtsp://user:pass@cam1.fritz.box:554/1/2
          roles:
            - detect
        - path: rtsp://user:pass@cam1.fritz.box:554/1/1
          roles:
            - record
      output_args:
        record: -f segment -segment_time 10 -segment_format mp4 -reset_timestamps 1 -strftime 1 -c:v copy -c:a aac
    detect:
      width: 640
      height: 360
      fps: 5
    objects:
      track:
        - person
        - umbrella
        - dog
        - cat
        - car
    motion:
      improve_contrast: true
      mask:
        - 0,0,444,0,420,82,240,135,0,202
    zones:
      zone_0:
        coordinates: 0,360,640,360,640,0,437,0,386,78,0,198
    snapshots:
      enabled: true
      timestamp: true
      quality: 95
      required_zones:
        - zone_0
      retain:
        default: 10
    record:
      enabled: true
      retain:
        days: 10
        mode: all
      events:
        post_capture: 10
        pre_capture: 10
        required_zones:
          - zone_0
        retain:
          default: 10
          mode: motion

**Reply by NickM-27** [reply]
Yeah, higher resolution would work better most likely, along with a slightly increased fps

**Reply by kuumaur** [reply]
640x360 is the highest resolution on sub stream.

**Reply by NickM-27** [reply]
Then you may just need to use the high res stream and scale it down to half size. That'll use more cpu but that's the recommendation for now I believe

**Reply by kuumaur** [reply]
Ok, changed to 7fps. I will test.

**Reply by AlexZorzi** [reply]
> Ok, changed to 7fps. I will test.

@kuumaur sorry to ping you after all this time, did you fix your issue?

**Reply by kuumaur** [reply]
> > Ok, changed to 7fps. I will test.
> 
> @kuumaur sorry to ping you after all this time, did you fix your issue?

I gave up and I have sent the camera back.

**Reply by AlexZorzi** [reply]
@kuumaur @NickM-27 

After a bit of debug it seems to be a limitation of the dataset used in the model, even tried more advanced models like yolov8, yolo11.

when the the images has a top to bottom view the model struggles to detect the object. 
while its very precise when the point of view is less vertical.
